src/components/Books/Books.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { SFC, useEffect, useState } from 'react';
+import IBook from '../../models/IBook';
+import { fetchBooks } from './api';
+import View from './BooksView';
```

Comment:
I was pretty interested to see what you would recommend here with my `Books` vs `Book`. I like this.